frontend/app/Lightbox/MyLightbox.jsx

```diff
@@ -157,7 +158,7 @@ class MyLightbox extends CommonSearchView {
             window.setTimeout(this.checkArchiveStatus, 3000);
         }).catch(err=>{
             console.error(err);
-            this.setState({showingArchiveSpinner: false, selectedRestoreStatus: err.response.data && err.response.data.detail ? err.response.data.detail : ""});
```

Comment:
does anything use `selectedRestoreStatus` when this is taken out? if not should remove it from the `state` definition in `constructor()`

Reply:
Yes.

Reply:
https://github.com/guardian/archivehunter/blob/da_new_icon/frontend/app/Lightbox/MyLightbox.jsx#L183

https://github.com/guardian/archivehunter/blob/da_new_icon/frontend/app/Lightbox/MyLightbox.jsx#L230

Reply:
👍